app/templates/verify.hbs

```diff
@@ -1,6 +1,10 @@
 {{#if this.success}}
   <div class="ui {{if this.isLoading 'loading'}} icon info message eight wide column center aligned">
-    <p>Your email has been verified successfully! Please login to continue.</p>
+    {{#if this.session.isAuthenticated}}
+      <p>Your email has been verified successfully!</p>
+    {{else}}
+      <p>Your email has been verified successfully! Please login to continue.</p>
+    {{/if}}
```

Comment:
More than half of the message is duplicated, please remove it

Reply:
Removed duplication! Please have a look now